.github/workflows/pulp_images.yml

```
@@ -108,14 +187,15 @@ jobs:
           ls -al $VOLUME_PATH
           tree $VOLUME_PATH
           http --follow --timeout 30 --check-status --pretty format --print hb http://localhost:8080/pulp/api/v3/status/ || true
-  galaxy:
+      - name: Display log on error
+        if: failure()
```

```suggestion
      - name: Display log
        if: always()
```
it is good to keep logging, if a dependency changes we can track by comparing the log from a successful run with the failed one

---

Yeah, or if the behavior is abnormal, but the tests pass.